3d/generate_2d.py

```diff
@@ -45,25 +46,29 @@
     svg_output = renderer.render_svgs()
 
     # Export to png
-    raster_svg = os.path.join(laser_parts_directory, 'raster.svg')
-    raster_png = os.path.join(laser_parts_directory, 'raster.png')
+    cwd = os.getcwd()
```

Comment:
Aha, that's really interesting! In that case this cwd usage seems find to me!

Another approach to try would be like I mentioned above with changing laser_parts_directory, but also throw in an `os.path.abspath`, so something like
`laser_parts_directory = os.path.abspath(os.path.join(os.path.dirname(__file__), 'build', 'laser_parts'))`.

Not a big deal though, I'm happy to merge as is.